### ISSUE TITLE
re-number figures to match episode numbers.

### ISSUE THREAD

**Issue by ilessing** [UCSBCarpentry/Reproducible-Publications-with-RStudio#65]
see episodes 6 and 7

```
_episodes/06-writing-styling.md:![Fig. 6.1 - Toolbar](../fig/07-toolbar.png)
_episodes/06-writing-styling.md:![Fig. 6.2 - Menu](../fig/07-menu.png)
_episodes/06-writing-styling.md:![Fig. 6.3 - Shortcuts](../fig/07-shortcuts.png)
_episodes/07-code-chunks.md:![blank Rmd code chunk](../fig/06-blank-code-chunk.PNG)
_episodes/07-code-chunks.md:![blank Rmd code chunk](../fig/06-blank-code-chunk.PNG)
_episodes/07-code-chunks.md:![heartrate code in chunk](../fig/06-heartrate-code.PNG)
_episodes/07-code-chunks.md:> ![auto create code chunk](../fig/06-auto-code-chunk.PNG)
_episodes/07-code-chunks.md:![code chunk with plot1 code](../fig/06-run-code-with-knit.PNG) 
_episodes/07-code-chunks.md:![run from code chunk](../fig/06-heartrate-code-runfromchunk.png)
_episodes/07-code-chunks.md:![run code menu](../fig/06-rmd-run-options.PNG)
_episodes/07-code-chunks.md:![heart rate code path error](../fig/06-HR-code-path-error.PNG)
_episodes/07-code-chunks.md:![code heart rate updated file path](../fig/06-HR-code-fixed-path.PNG)
_episodes/07-code-chunks.md:![Heart rate code no options for code chunk](../fig/06-HR-output-no-options.PNG)
_episodes/07-code-chunks.md:>> ![solution to 9.3](../fig/06-solution-9.3.PNG)    
_episodes/07-code-chunks.md:![basic code chunk](../fig/06-blank-code-chunk.PNG)
_episodes/07-code-chunks.md:![Fig 4 path error](../fig/06-fig4-connection-error.PNG)
_episodes/07-code-chunks.md:![change working directory rmd file](../fig/06-change-wd.PNG)
_episodes/07-code-chunks.md:![load libraries & data](../fig/06-load-libraries-data.PNG)

```

**Comment by ilessing** [comment]
addressed with d0eed3f and a466725

**Comment by ilessing** [comment]
I'm closing as Fixed.
